src/pysdmx/io/xml/__structure_aux_reader.py

```diff
@@ -551,8 +551,8 @@ def __format_con_id(self, concept_ref: Dict[str, Any]) -> Dict[str, Any]:
         return rep
 
     @staticmethod
-    def __format_relationship(json_rel: Dict[str, Any]) -> Optional[str]:
-        att_level = None
+    def __format_relationship(json_rel: Dict[str, Any]) -> str:
```

Comment:
If this method is meant to return an attachment level, maybe the name __format_relationship is confusing? What about something like __get_attachment_level instead?